modules/plot/main/coffee/charts/PieChart.coffee

```diff
@@ -79,7 +81,7 @@ class PieChart extends hx.EventEmitter
 
   render: ->
     self = this
-    selection = hx.select(@selector)
+    selection = hx.select(@selector).class('hx-pie-chart')
```

Comment:
Thought about this and I'm not sure this is something we want to add really as it breaks the idea behind how graphs and charts work currently. Generally, you have to either set explicit heights for each individual graph/chart or you have a class that sets min/max values.

I think it would be better in this case to add:
```
hx.pieChart(options).style('height', '500px').style('width','200px')
hx.pieChart(options).class('myPieClass')
```

Combining this with the resize events in this PR would resolve the issues with the pie chart fluid api